### PR TITLE
add repeat, alphasort entries

### LINKED ISSUE #1328: "repeat" behavior
Dave Beaudet, in #1264:
> consider use case where you'd have an in-exhibition viewer that cycles through via auto-advance and upon reaching the end, starts over from the beginning.

I think that this is a separate behavior from "auto-advance" and needs its own identity.  I could imagine both cases where you want to repeat from the first canvas (as above), and situations where you don't (watching a multi-canvas video).

Propose:  "repeat" value for `behavior`. When this value and "auto-advance" are present, the client _SHOULD_ continue from the last canvas back to the first once the last canvas has been completed.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #1328

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -483,16 +483,17 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
...
-| `paged` | Valid on Manifest, Sequence and Range, which include Canvases that have at least `height` and `width` dimensions. Canvases included in resources with this `behavior` represent pages in a bound volume, and _SHOULD_ be presented in a page-turning interface if one is available.  The first canvas is a single view (the first recto) and thus the second canvas likely represents the back of the object in the first canvas. If this is not the case, see the "non-paged" `behavior`. |
+| `auto-advance` | Valid on Collection, Manifest, Sequence and Canvas, that include or are Canvases with at least the `duration` dimension. When the client reaches the end of a Canvas with a duration dimension that has (or is within a resource that has) this `behavior`, it _SHOULD_ immediately proceed to the next Canvas and render it. If there is no subsequent Canvas in the current context, then this `behavior` should be ignored. When applied to a Collection, the client should treat the first Canvas of the next Manifest as following the last Canvas of the previous Manifest, respecting any `startCanvas` specified. |
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: > Valid on Collection, Manifest, Sequence and Canvas, that include or are Canvases

Typo? Shouldn't "Collection, Manifest, Sequence and Canvas" each be pluralized? We use this construction elsewhere as well and it reads awkwardly (e.g. `repeat`, `paged`)

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -483,16 +483,17 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
...
 | `continuous` | Valid on Manifest, Sequence and Range, which include Canvases that have at least `height` and `width` dimensions.  Canvases included in resources with this `behavior` are partial views and an appropriate rendering might display all of the Canvases virtually stitched together, such as a long scroll split into sections. This `behavior` has no implication for audio resources. The `viewingDirection` of the Sequence or Manifest will determine the appropriate arrangement of the Canvases. |
+| `facing-pages` | Valid only for Canvas, where the Canvas has at least `height` and `width` dimensions. Canvases with this `behavior`, in a Sequence or Manifest with the "paged" `behavior`, _MUST_ be displayed by themselves, as they depict both parts of the opening.  If all of the Canvases are like this, then page turning is not possible, so simply use "individuals" instead. |
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: I don't think "behavior" should be in backtics here -  we're talking about _a_ behavior, not the _keyword_ `behavior`, right?

Comment (jpstroop) [reply source/api/presentation/3.0/index.md]: Applies below as welll

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -483,16 +483,17 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
...
+| `individuals` | Valid on Collection, Manifest, Sequence and Range. For Collections with this `behavior`, each of the included Manifests are distinct objects. For Manifest, Sequence and Range, the included Canvases are distinct views, and _SHOULD NOT_ be presented in a page-turning interface. This is the default `behavior` if none are specified. |
 | `multi-part` | Valid only on Collection. Collections with this `behavior` consist of multiple Manifests that each form part of a logical whole, such as multi-volume books or a set of journal issues. Clients might render the Collection as a table of contents, rather than with thumbnails. |
+| `no-nav` | Valid only for Range. Ranges with this `behavior` _MUST NOT_ be displayed to the user in a navigation hierarchy. This allows for Ranges to be present that capture unnamed regions with no interesting content. |
```
Comment (jpstroop) [inline source/api/presentation/3.0/index.md]: > with no interesting

Maybe explain a little more by adding a use case?

File: source/api/presentation/3.0/index.md (hunk 2 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: This is #1330, I think.  Or at least should be part of that!

File: source/api/presentation/3.0/index.md (hunk 1 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: :+1: I thought this at the time, but it wasn't part of the issue so I didn't change it.  Changing :)

File: source/api/presentation/3.0/index.md (hunk 2 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: (Leaving it for that discussion, have made a note of this there -- so wontfix yet)

File: source/api/presentation/3.0/index.md (hunk 3 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: It's the extent vs nav structure case.  This allows boring content to be part of a sub-range without polluting a nice clean ToC with it.  Will try to explain succinctly.

**Pull request merged**